### ISSUE TITLE
Save user symptoms during onboarding

### ISSUE THREAD

**Issue by pyritewolf** [instedd/oasis#12]
- [x] Add a Symptoms model to the database & story x syptoms relation (a user's story can show many symptoms)
- [x] Add initial seed script
- [x] Add `timestamps` columns to story_symptoms
- [x] Add GET & POST endpoints with the same degree of security as story endpoints
- [x] Save symptoms from the frontend, after the onboarding of a sick person

**Comment by lmatayoshi** [comment]
I'm thinking about using these simple models:

```
symptoms:
  id: integer
  name: string
  
stories_symptoms:
  story_id: integer (FK),
  symptom_id: integer (FK),
  PRIMARY KEY(story_id, symptom_id)
```

**Comment by nditada** [comment]
Please consider that it is likely that we will need to capture symptoms multiple times over time for a single individual.

**Comment by anaPerezGhiglia** [comment]
closed by 497c088
